torch/lib/c10d/logger.cpp

```diff
@@ -11,6 +11,8 @@ namespace {
 
 const int kMilliSecondToNanosSecond = 1000000;
 
+const char * kDefaultDDPComm = "builtin_allreduce";
```

Comment:
This name can be confusing, since there is a built-in hook called AllReduce, but this value here actually means no hook, when allreduce is used w/o triggering a comm hook.

It will be better to use an empty string to represent the default value, indicating that no hook is registered.

Reply:
Sounds good, will make the empty string the default/no comm hook registered value